### DISCUSSION TITLE
React Next.js Example

### DISCUSSION THREAD

**Discussion by tradex-app** [tradex-app/TradeX-chart#31] [Announcements]
The repo now has a nextjs directory, with an example of the chart implemented with React and Next.js.
https://github.com/tradex-app/TradeX-chart/tree/master/nextjs

**Comment by ramachoon** [comment]
Thank you for sharing the update on the repository. I have reviewed the nextjs directory and the example of the chart implemented with React and Next.js. It looks great!

I appreciate your effort in adding this feature to the TradeX-chart repository. If there are any specific areas or functionalities you would like me to focus on or if you have any further instructions, please let me know. I'm here to assist you.

Once again, thank you for the update, and I look forward to continuing our work on this project.

Best regards,
Ramachandran.

**Reply by tradex-app** [reply]
You're most welcome Ramachandran. Thanks must go to the other contributors who provided this. There will be further updates coming.
